src/components/about/Intro.js

```diff
@@ -117,11 +117,6 @@ const Intro = () => {
           </Grid>
         ))}
       </Grid>
-      <Grid className={buttonContainer}>
```

Comment:
We can remove `Link`, `Button` and `getPageUrl` from this file.

Like this
import React from 'react';
<s style="color: red">import { Link } from 'gatsby';</s>
import { Grid, Typography, Hidden, <s>Button</s> } from '@material-ui/core';
import { makeStyles } from '@material-ui/styles';

<s>import { getPageUrl } from '../../Routes';</s>
import { spacing, menuHeightOffset, colors, stylesBase } from '../styledComponents';
import hero from '../../assets/aboutPage/hero.jpg';
import heroMobile from '../../assets/aboutPage/hero.mobile.jpg';
import ourVisionBackground from '../../assets/aboutPage/ourVisionBackground.jpg';
import ourMissionBackground from '../../assets/aboutPage/ourMissionBackground.jpg';